src/TCL.xml

```diff
@@ -7,10 +7,9 @@
       </crossRefs>
     <text>
       <copyrightText>
-         <p>This software is copyrighted by
-        <alt match=".+" name="authors">the Regents of the University of California, Sun Microsystems, Inc.,
-        Scriptics Corporation, ActiveState Corporation</alt> and other parties. The following terms apply to all
-        files associated with the software unless explicitly disclaimed in individual files.
+         <p>This software is copyrighted by the Regents of the University of California, Sun Microsystems, Inc.,
+        Scriptics Corporation, ActiveState Corporation and other parties. The following terms apply to all
+        files associated with the software unless explicitly disclaimed in individual files. 
```

Comment:
I dont' think the sentence beginning with "The following terms" should be part of the copyright notice and tag

Reply:
I agree - just pushed a fix

Reply:
so, now the beginning <p> tag is before the copyrighttext tag, and end </p> tag after the bit of text. not sure it matters, but that seems a bit weird?

Reply:
The paragraph tag is used only for formatting.  You can't have a paragraph start tag outside the copyright tag and the end tag inside the copyright tag.  So, to keep the formatting the same as the original, I moved the start paragraph tag outside the copyrighttext tag and left the end tag where it was.

Reply:
i see. just looks weird, but fine by me!